src/sentry/models/orgauthtoken.py

```diff
@@ -151,10 +152,21 @@ def get_org_auth_token_id_from_auth(auth: object) -> int | None:
 def update_org_auth_token_last_used(auth: object, project_ids: list[int]):
     org_auth_token_id = get_org_auth_token_id_from_auth(auth)
     organization_id = getattr(auth, "organization_id", None)
-    if org_auth_token_id is not None and organization_id is not None:
-        orgauthtoken_service.update_orgauthtoken(
-            organization_id=organization_id,
-            org_auth_token_id=org_auth_token_id,
-            date_last_used=timezone.now(),
-            project_last_used_id=project_ids[0] if len(project_ids) > 0 else None,
-        )
+    if not org_auth_token_id and not organization_id:
```

Comment:
Why are we no longer checking these values explicitly against `None`, like we are in the previous version? The behavior here could be different if either value is false-y but not `None`; is this behavior change intentional?

Reply:
Projects and orgs cannot have id values of 0. I'll put the none checks back though.